zos.json

```diff
@@ -3,7 +3,7 @@
   "name": "UpgradableAssetToken",
   "version": "0.1.0",
   "contracts": {
-    "UpgradeableAssetToken": "UpgradeableAssetToken"
+    "AssetToken": "UpgradeableAssetToken"
```

Comment:
I would move the code from UpgradeableAssetToken into AssetToken, then delete UpgradeableAssetToken. we don't plan on supporting two versions of the asset token